go.mod

```diff
@@ -46,7 +46,7 @@ require (
 	github.com/DataDog/zstd v1.4.5 // indirect
 	github.com/Masterminds/semver/v3 v3.1.0
 	github.com/Masterminds/sprig/v3 v3.1.0
-	github.com/aws/aws-sdk-go v1.33.5
```

Comment:
Could we split the go mod changes + generated changes into a separate commit(s)?